src/test/java/com/google/cloud/tools/gradle/appengine/core/CloudSdkDownloaderTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2018 Google Inc. All Right Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.cloud.tools.gradle.appengine.core;
+
+import com.google.cloud.tools.managedcloudsdk.BadCloudSdkVersionException;
+import com.google.cloud.tools.managedcloudsdk.OsInfo;
+import com.google.cloud.tools.managedcloudsdk.OsInfo.Name;
+import com.google.cloud.tools.managedcloudsdk.UnsupportedOsException;
+import com.google.cloud.tools.managedcloudsdk.Version;
+import com.google.cloud.tools.managedcloudsdk.components.SdkComponentInstaller;
+import com.google.cloud.tools.managedcloudsdk.install.SdkInstaller;
+import com.google.cloud.tools.managedcloudsdk.update.SdkUpdater;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.gradle.internal.impldep.org.junit.Rule;
+import org.gradle.internal.impldep.org.junit.Test;
+import org.junit.Assert;
+import org.junit.rules.TemporaryFolder;
+
+public class CloudSdkDownloaderTest {
+
+  @Rule TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  private CloudSdkDownloader downloader;
```

Comment:
It looks like this field can just be a local variable, or just set to `new CloudSdkDownloader()`.